<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AS400ConfigWidget</class>
 <widget class="QWidget" name="AS400ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="driverLabel">
     <property name="text">
      <string>AS400 ODBC Driver</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="driverLineEdit">
     <property name="text">
      <string>iSeries Access ODBC Driver</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="systemLabel">
     <property name="text">
      <string>AS400 Hostname or IP</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="systemLineEdit">
     <property name="text">
      <string>Insert system IP or hostname here</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="usernameLabel">
     <property name="text">
      <string>AS400 Username</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="usernameLineEdit">
     <property name="text">
      <string>Insert username here</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>AS400 Password</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="text">
      <string>password</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>64</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="commitSettings">
     <property name="text">
      <string>Commit Settings</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
